src/routeApp/docs.js

```diff
@@ -0,0 +1,182 @@
+/**
+ * @openapi
+ * tags:
+ *  name: Routes
+ *  description: Routes for the routes app
+ */
+
+// Security Schemes
+/**
+ * @openapi
+ * components:
+ *  securitySchemes:
+ *    Token:
+ *      type: http
+ *      scheme: Bearer
+ *      bearerFormat: JWT
+ *      required: true
+ */
+
+// -------------------//-------------------//
+
+/**
+ * @swagger
+ * /api/v1/routes:
+ *  post:
+ *      security:
+ *         - Token: []
+ *      summary: Add a new route in the database
+ *      description: this request will take in route info to be created and add the driver to the database or return specific error
+ *      tags:
+ *          - Routes
+ *      requestBody:
+ *          required: true
+ *          content:
+ *              application/json:
+ *                  schema:
+ *                      $ref: "#/components/schemas/Route"
+ *      responses:
+ *          201:
+ *              description: Return a created route
```

Comment:
Remember to add the response for status `401 and 403` respectively for Unauthorized and denied access